lib/rucio/api/replica.py

```diff
@@ -242,3 +243,17 @@ def get_suspicious_files(rse_expression, younger_than=None, nattempts=None):
     :param nattempts: The number of time the replicas have been declared suspicious
     """
     return replica.get_suspicious_files(rse_expression=rse_expression, younger_than=younger_than, nattempts=nattempts)
+
+
+def set_tombstone(rse, scope, name, issuer):
+    """
+    Sets a tombstone on one replica.
+
+    :param rse: name of the RSE.
+    :param scope: scope of the replica DID.
+    :param name: name of the replica DID.
+    :param issuer: The issuer account
+    """
+    if not permission.has_permission(issuer=issuer, action='set_tombstone', kwargs={}):
```

Comment:
We should add the permission (just check against admin) to the permission files too.

Reply:
I wanted to use the default permission check: [https://github.com/rucio/rucio/blob/master/lib/rucio/core/permission/generic.py#L111](https://github.com/rucio/rucio/blob/master/lib/rucio/core/permission/generic.py#L111).
Should I add it anyway?

Reply:
Oh yes right, just leave it like this then 😄 Sorry for the noise.